storage/src/lib.rs

```diff
@@ -71,71 +88,81 @@ pub fn aleo_ledger_dir(network: u16, dev: Option<u16>) -> PathBuf {
 }
 
 ///
-/// Returns the directory for accessing the operator files from Aleo storage.
+/// Returns the directory for accessing the prover files from Aleo storage.
 ///
-/// In production mode, the expected directory path is `~/.aleo/storage/operator-{network}`.
-/// In development mode, the expected directory path is `/path/to/repo/.operator-{network}-{id}`.
+/// In production mode, the expected directory path is `~/.aleo/storage/prover-{network}`.
+/// In development mode, the expected directory path is `/path/to/repo/.prover-{network}-{id}`.
```

Comment:
maybe we can move both the ledger and the BFT-related storage to a new `../repo/.storage` path :thinking:?